code/modules/research/techweb/all_nodes.dm

```diff
@@ -481,7 +481,7 @@
 	display_name = "Cybernetic Organs"
 	description = "We have the technology to rebuild him."
 	prereq_ids = list("adv_biotech")
-	design_ids = list("cybernetic_heart", "cybernetic_liver", "cybernetic_lungs")
+	design_ids = list("cybernetic_ears", "cybernetic_heart", "cybernetic_liver", "cybernetic_lungs", )
```

Comment:
superfluous comma?

Reply:
This comma might actually cause issues. If I remember correctly, this adds a null to the list.